### PR TITLE
Fix information about showed entries below paginated table. Closes #R…

### PR DESCRIPTION
**Pull request by lukas-vlcek**
…HD-532.

### CODE REVIEW

**Comment by paulrobinson** [comment]
There was a build for this: http://developers-pr.stage.redhat.com/pr/297/build/588/. Not sure why the reporting on here was lost.

**Comment by paulrobinson** [comment]
@wesbos can you review? I agree with @lukas-vlcek's comment that it's probably better to move the calculations into the controller. Thoughts?

**Comment by wesbos** [comment]
:+1:  good to go from a code pov

File: javascripts/developer-materials.angular.js [hunk 1]
```diff
@@ -243,6 +255,12 @@ dcp.controller('developerMaterialsController', function($scope, materialService)
     size : 10
   };
 
+  // Add Math object to $scope so we can use it directly in Angular expressions
+  // like: {{ Math.min(data.materials.length, paginate.currentPage * pagination.size) }}
+  // This might not be clean technique from Angular perspective (more clear would be
+  // to do all required calculations in controller and not the view)
+  $scope.Math = Math;
```
Comment (wesbos) [inline javascripts/developer-materials.angular.js]: Goign to fix this - Math should not be passed into the scope

**Pull request merged**